src/UKHO.ERPFacade.API/Controllers/ErpFacadeController.cs

```diff
@@ -28,27 +28,27 @@ public class ErpFacadeController : BaseController<ErpFacadeController>
 
         [HttpPost]
         [Route("/erpfacade/priceinformation")]
-        public virtual async Task<IActionResult> Post([FromBody] JObject requestJson)
+        public virtual async Task<IActionResult> PostPriceInformation([FromBody] JArray requestJson)
         {
-            var traceId = requestJson.SelectToken(TraceIdKey)?.Value<string>();
+            var corrId = requestJson.First.SelectToken(CorrIdKey)?.Value<string>();
 
-            if (string.IsNullOrEmpty(traceId))
+            if (string.IsNullOrEmpty(corrId))
             {
-                _logger.LogWarning(EventIds.TraceIdMissingInSAPEvent.ToEventId(), "TraceId is missing in the event received from the SAP.");
+                _logger.LogWarning(EventIds.TraceIdMissingInSAPEvent.ToEventId(), "CorrId is missing in the event received from the SAP.");
```

Comment:
Can this be renamed as CorrIdMissinginSAPEvent

Reply:
Updated